napari/utils/_base.py

```diff
@@ -7,8 +7,19 @@
 
 from appdirs import user_config_dir
 
+from napari._version import __version_tuple__
+
+if 'dev' in str(__version_tuple__):
+    _dev_version_config_dir = 'dev'
+else:
+    _dev_version_config_dir = None
```

Comment:
```suggestion
from packaging import version

from napari._version import __version__

_dev_version_config_dir = 'dev' if version.parse(__version__).is_devrelease else None

```

Maybe this version. Your code will not work as version tuple will contain `dev{num}` like `dev18`, not simple `dev`.

Reply:
my string checking did work because
```
if 'dev' in 'dev28':
     print("Yah")
```
does print, but your way is of course way more elegant.

Reply:
Yes, but you check presence in `__version_tuple__` not, `__version__`
```
__version_tuple__
Out[3]: (0, 4, 17, 'dev73', 'g2e32f34f9.d20221107')
"dev" in __version_tuple__
Out[4]: False
```